packages/backend/migrations/20240916181643_require_estimated_exchange_rate.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+exports.up = function (knex) {
+  return knex('quotes')
+    .whereNull('estimatedExchangeRate')
```

Comment:
In which cases the `estimatedExchangeRate` was null?

Reply:
I dont know of any specific cases where it would be null, but since it's nullable we should handle that possibility.